pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -350,6 +350,12 @@ func (lb *ListenerBuilder) buildHTTPProxyListener(configgen *ConfigGeneratorImpl
 }
 
 func (lb *ListenerBuilder) buildVirtualOutboundListener(configgen *ConfigGeneratorImpl) *ListenerBuilder {
+	noneMode := lb.node.GetInterceptionMode() == model.InterceptionNone
+	if noneMode {
```

Comment:
Nit: the two lines can be merged 

Except for this looks good to me.

Reply:
merged